sections/home-hero-section/index.js

```diff
@@ -0,0 +1,139 @@
+import { rem } from 'polished'
+import Typist from 'react-typist'
+import styled from 'styled-components'
+
+import { ApplyLink, Heading, Text } from 'components'
+import 'react-typist/dist/Typist.css'
+import { TABLET_LARGE_SIZE, MOBILE_SIZE } from 'styles/constants'
+import styles from 'styles/Home.module.css'
```

Comment:
Each section should define its own styles. `Home.module.css` should only be used in `index.js`. 
Might actually be better to define the required styles via styled components to for consistency.

Reply:
Removed reference to Home.module.css and am using styled components as suggested.